tests/tokens.php

```diff
@@ -1,7 +1,3 @@
 <?php
 
-$db_tokens_map = array(
-    'Test' => 'NeverGonnaGiveYouUp'
-);
-
-$db_tokens = array_values($db_tokens_map);
\ No newline at end of file
+$db_tokens = array("NeverGonnaGiveYouUp");

```

Comment:
I don't like that, I keep maps so that I can identify which token is used where and better know what tokens to revoke.

On top of being useless, this modification destroys meaning